notifique/src/main/java/com/nathanrassi/notifique/NotifiqueListView.kt

```diff
@@ -224,7 +223,13 @@ internal class NotifiqueListView(
 
     internal fun setNotifique(notifique: Notifique) {
       appName.text = notifique.app
-      timestamp.text = timestampFormat.format(Date(notifique.timestamp))
+      var midnightCalendar = Calendar.getInstance(Locale.US)
+      midnightCalendar.set(Calendar.HOUR, 12); midnightCalendar.set(Calendar.MINUTE, 0);midnightCalendar.set(Calendar.SECOND, 0);midnightCalendar.add(Calendar.DAY_OF_MONTH, -1) //var notificationTime = Date(notifique.timestamp)
+      if (Date(notifique.timestamp).after(midnightCalendar.time)) {
+        timestamp.text = timestampFormat.format(Date(notifique.timestamp))
+      } else {
+        timestamp.text = dateTimestampFormat.format(Date(notifique.timestamp))
```

Comment:
there's a lot to be said about the code cleanup, but i think there's a larger issue.
isn't there a standard Android API for displaying dates in the format you want? probably not in java.util, maybe in java.date, but probably in the Android APIs.
i'll look.